docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1346,6 +1346,19 @@ Query Parameters:
 -   **memswap** - Total memory (memory + swap), `-1` to disable swap.
 -   **cpushares** - CPU shares (relative weight).
 -   **cpusetcpus** - CPUs in which to allow execution (e.g., `0-3`, `0,1`).
+-   **buildargs** – JSON map of string pairs for build-time
```

Comment:
By trying to explain everything here in this parameter description, you really just end up overloading the newbiew reader. Most folks  using an API probably already know what they are and don't need the detail. Xref is enough.

---> https://gist.github.com/moxiegirl/68562de671c376368666

-   **buildargs** – JSON map of string pairs for build-time variables. Users can
set these values at build-time and they are used as environment context for the
command(s) run as part of the Dockerfile's `RUN` primitive. Read more about `ARG` primitive  [here](https://docs.docker.com/reference/builder/#arg)